src/deprecated/ActionMenu.tsx

```diff
@@ -18,7 +18,7 @@ interface ActionMenuBaseProps extends Partial<Omit<GroupedListProps, keyof ListP
   /**
    * A callback that triggers both on clicks and keyboard events. This callback will be overridden by item level `onAction` callbacks.
    */
-  onAction?: (props: ItemProps, event?: React.MouseEvent<HTMLDivElement> | React.KeyboardEvent<HTMLDivElement>) => void
+  onAction?: (props: ItemProps, event?: React.MouseEvent<HTMLLIElement> | React.KeyboardEvent<HTMLLIElement>) => void
```

Comment:
Could these type changes break anything?

Reply:
I don't expect any consumers will have coded any checks for these types, but if they have I think they should get the warning as the elements have changed.